doc/libpmemobj/pmemobj_ctl_get.3.md

```diff
@@ -199,6 +199,10 @@ heap.thread.arena_id | r- | - | unsigned | - | - | -
 
 Reads the index of the arena assigned to the current thread.
 
+heap.arena.create | --x | - | - | - | - | -
+
+Create and initialize one new arena in the heap.
```

Comment:
`Creates and initializes a new arena in the heap.`


You need to have at least one output: the index of the newly created arena.